.github/ISSUE_TEMPLATE/compose_feature.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Add all ISSUE_TEMPLATE files in a separate PR

Reply:
done